lib/tapioca/gem/listeners/yard_doc.rb

```diff
@@ -64,7 +66,13 @@ def documentation_comments(name, sigs: [])
 
           comments = docstring.lines
             .reject { |line| IGNORED_COMMENTS.any? { |comment| line.include?(comment) } }
-            .map! { |line| RBI::Comment.new(line) }
+            .map! do |line|
+              if line.strip.start_with?(RBS_SIGNATURE_PREFIX)
+                RBI::RBSSig.new(line[1..])
```

Comment:
Considering `#: -> void`, we already ignore the `#` fine, we also skip the `:`, but we should strip the result before passing it to `RBSSig.new`.